documentation/pages/move-registry/maintainer-practices.mdx

```diff
@@ -0,0 +1,25 @@
+import Beta from '../../components/beta'
+
+<Beta/>
+
+# Best Practices
+
+For package authors and maintainers, the following best practices are recommended for working with the Move Registry. Following these tips ensures that other builders can use your package as intended.
+
+## Automated Address Management
+
+- Whenever you publish or upgrade a package, use the Sui Automated Address Management feature.
```

Comment:
Can we link to the AAM concept page here as well?

Reply:
The link that follows is for the whole section. Used its own sentence instead of just the title so can tell reader they are opening another site.